_apps/yamburger.md

```diff
@@ -0,0 +1,36 @@
+---
+title: YAMBURGER
+description: YAMBURGER finds YAML syntax errors (or YAMBURGERS) on files changed in GitHub Pull Requests.
```

Comment:
> YAMBURGER finds YAML syntax errors (or YAMBURGERS) on files changed in GitHub Pull Requests.

Could this become:

> Finds YAML syntax errors (or YAMBURGERS) on files changed in GitHub Pull Requests.

The way the website displays the app names above the description would make this look rather repetitive.

Reply:
> The way the website displays the app names above the description would make this look rather repetitive.

Great suggestion @hiimbex! I've applied it in 631c038.